src/test/java/io/zeebe/exporter/proto/RecordTransformTest.java

```diff
@@ -806,4 +811,92 @@ private void assertMetadata(
 
     return record;
   }
+
+
+  @Test
+  public void shouldUnpackAllRecordTypes() throws Exception {
+
+    // given
+    
+    Map<Class<? extends Message>, Record<?>> records = new HashMap<>();
+
+    final DeploymentRecordValue deploymentRecordValue = mockDeploymentRecordValue();
+    final Record<DeploymentRecordValue> deploymentRecord = mockRecord(deploymentRecordValue, ValueType.DEPLOYMENT, DeploymentIntent.CREATE);
+    records.put(Schema.DeploymentRecord.class, deploymentRecord);
```

Comment:
The test case is a bit hard to read because of all the different test parameters. We could make it more clear by transforming it into a parameterized test.